Transcendence/PlayMakerExtensions.cs

```diff
@@ -63,7 +64,9 @@ internal static void PrependAction(this FsmState s, Action a)
 
         internal static void ReplaceAction(this FsmState s, int i, Action a)
         {
-            s.Actions[i] = new FuncAction(a);
+            FuncAction action = new FuncAction(a);
```

Comment:
I'll fix this myself on merging, but I usually use `var` to declare most variables; keep that in mind for future contributions here.

Reply:
I'll keep that in mind.